templates/vue/src/components/SettingsModal.vue

```diff
@@ -98,23 +193,90 @@ export default {
         backgroundUrl = "",
         autoLayout = false,
         nodeDraggable = true,
+        defaultPermissions = {
+          public: ["read"],
+          authenticated: ["read"],
```

Comment:
Is this serving a purpose? If you are setting it here, you shouldn't set it when declaring it (or vise-versa).